### ISSUE TITLE
Group files are malformed

### ISSUE THREAD

**Issue by dmtucker** [Isilon/isilon_hadoop_tools#28]
They always lead with a comma. For example:
```
hadoop-test:x:1027:,hdfs-test,mapred-test,yarn-test,HTTP-test,hbase-test
```
* Note the `,hdfs-test`.

I've been told "some distro might tolerate it but don’t do it".

**Comment by bonibruno** [comment]
Fixed.  See PR bonibruno-patch3.

**Comment by bonibruno** [comment]
Fixed in PR #32, there is a comma at the end, but this is fine.
